configs/temporal.json

```diff
@@ -7,34 +7,114 @@
     },
     {
       "url": "https://docs.temporal.io/docs/concepts/workflows",
-      "page_rank": 998
+      "page_rank": 999
     },
     {
       "url": "https://docs.temporal.io/docs/concepts/activities",
-      "page_rank": 997
+      "page_rank": 999
     },
     {
       "url": "https://docs.temporal.io/docs/concepts/events",
-      "page_rank": 996
+      "page_rank": 999
     },
     {
       "url": "https://docs.temporal.io/docs/concepts/queries",
-      "page_rank": 995
+      "page_rank": 999
     },
     {
       "url": "https://docs.temporal.io/docs/concepts/task-queues",
+      "page_rank": 999
+    },
+    {
+      "url": "https://docs.temporal.io/docs/glossary",
+      "page_rank": 999
+    },
+    {
+      "url": "https://docs.temporal.io/docs/server/introduction",
+      "page_rank": 998
+    },
+    {
+      "url": "https://docs.temporal.io/docs/server/quick-install",
+      "page_rank": 998
+    },
+    {
+      "url": "https://docs.temporal.io/docs/server/production-deployment",
+      "page_rank": 998
+    },
+    {
+      "url": "https://docs.temporal.io/docs/server/server-architecture",
+      "page_rank": 998
+    },
+    {
+      "url": "https://docs.temporal.io/docs/server/versions-and-dependencies",
+      "page_rank": 998
+    },
+    {
+      "url": "https://docs.temporal.io/docs/server/namespaces",
+      "page_rank": 998
+    },
+    {
+      "url": "https://docs.temporal.io/docs/server/workflow-search",
+      "page_rank": 998
+    },
+    {
+      "url": "https://docs.temporal.io/docs/server/elasticsearch-setup",
+      "page_rank": 998
+    },
+    {
+      "url": "https://docs.temporal.io/docs/server/options",
+      "page_rank": 998
+    },
+    {
+      "url": "https://docs.temporal.io/docs/server/security",
+      "page_rank": 998
+    },
+    {
+      "url": "https://docs.temporal.io/docs/server/configuration",
+      "page_rank": 998
+    },
+    {
+      "url": "https://docs.temporal.io/docs/server/archive-data",
+      "page_rank": 998
+    },
+    {
+      "url": "https://docs.temporal.io/docs/go/introduction",
+      "page_rank": 997
+    },
+    {
+      "url": "https://docs.temporal.io/docs/java/introduction",
+      "page_rank": 996
+    },
+    {
+      "url": "https://docs.temporal.io/docs/php/introduction",
+      "page_rank": 995
+    },
+    {
+      "url": "https://docs.temporal.io/docs/node/introduction",
+      "page_rank": 994
+    },
```

Comment:
hopefully this pagerank propagates to the rest of the pages, trying to be lazier here as these will evolve more

Reply:
All URLs under the given `url` will have the following `page_rank`!

Reply:
thanks as always shortcuts